sympy/core/tests/test_numbers.py

```diff
@@ -1749,3 +1748,75 @@ def test_Integer_precision():
     assert Float('1.0', precision=Integer(15))._prec == 15
     assert type(Float('1.0', precision=Integer(15))._prec) == int
     assert sympify(srepr(Float('1.0', precision=15))) == Float('1.0', precision=15)
+
+def test_numpy_to_float(): 
+    def check_approximate_equality(numpyval, testnum):
+            fromnumpy = Float(numpyval)
+            fromS = Float(S(testnum), precision = fromnumpy._prec)
+            assert (np.finfo(numpyval).nmant == fromnumpy._prec)
+            assert(abs(fromnumpy-fromS)/fromS<2**(-fromnumpy._prec+1))
+
+    from sympy.external import import_module
+    np = import_module('numpy')
+    testnum = 123456789
+
+    a1 = np.float32(testnum)
+    check_approximate_equality(a1, testnum)
+    a2 = np.float64(testnum)
+    check_approximate_equality(a2, testnum)
+    a3 = np.float128(testnum)
+    check_approximate_equality(a3, testnum)
+
+    fromnumpy = Float(a3)
+    fromS = Float(S(testnum), precision = fromnumpy._prec)
```

Comment:
`kwarg=a` without space for keyword arguments.  With space for assignment: `fromS = Float...` is correct.

Reply:
Thank you for explaining - fixed.